src/translators/crate.py

```diff
@@ -54,9 +54,12 @@ def setup(self):
         url = "{}:{}".format(self.host, self.port)
         self.ccm = ConnectionManager()
         self.connection = self.ccm.get_connection('crate')
+        # Added backoff_factor for retry interval between attempt of consecutive retries
+        backoff_factor = EnvReader(log=logging.getLogger(__name__).debug) \
+            .read(FloatVar('CRATE_BACKOFF_FACTOR', 0.0))
         if self.connection is None:
```

Comment:
@daminichopra I refactored a bit your code to use facilities included in `utils.cfgreader`. keep this library in mind in case in future you need to read env variables.